package.json

```diff
@@ -6,15 +6,15 @@
     "build:addon": "rollup -c && web-ext build --overwrite-dest && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi",
     "build:schema": "node schemas/generate-schema.mjs",
     "build:storybook": "build-storybook -s ./",
-    "dev": "rollup -c --config-enable-developer-mode",
+    "dev": "npm run build:schema && rollup -c --config-enable-developer-mode",
     "doc": "jsdoc -c jsdoc.conf.json && mkdir -p docs/images && cp images/rally-logo.png docs/images/",
     "lint": "npm-run-all lint:web-ext lint:eslint",
     "lint:eslint": "eslint .",
     "lint:web-ext": "web-ext lint",
     "package": "npm run build && web-ext build --overwrite-dest --ignore-files \"./tests/**/*\" --ignore-files \"./src/**/*\" --ignore-files \"./rollup.config.js\"",
     "start": "npm run build && web-ext run",
     "test:unit": "jest",
-    "test:integration": "npm run package && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi && mocha --timeout 30000 \"./tests/integration/*.js\"",
+    "test:integration": "npm run dev && web-ext build --overwrite-dest && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi && mocha --timeout 30000 \"./tests/integration/*.js\" 2>&1 | tee integration.log",
```

Comment:
After much investigation, I think that capturing `stdout` and then parsing it ourselves is the best we're going to get anytime soon with Firefox. I'd rather not do it here though, ideally, I'd like Selenium to log Geckodriver output (which it invokes) so we aren't trying to read the log that is currently being written to. I'm a little worried about things like buffering causing us problems, although I haven't seen it so far.